_services/strategy.md

```diff
@@ -4,15 +4,13 @@ title: Strategy / Consulting
 slug: strategy
 weight: 1
 summary: |
-  Simple execution is never simple until all avenues have been explored. A stitch in time saves nine, and a minute of planning saves 10 in execution. We've learned these lessons and cherish a challenge to think creatively for the most appropriate solutions for our partners.
+  Our seasoned team combines wisdom from years of digital consulting experience with a holistic eye to think creatively, leaving no stone unturned to discern the best solutions for our partners.
```

Comment:
Pairing up something really strong like "wisdom from years of digital consulting experience" with "a holistic eye to think creatively" isn't working...I think because eyes don't think creatively, and because you're comparing our collective wisdom with a single thing that we can't share (eye). I'd change to:

Our seasoned team combines wisdom from years of digital consulting experience with a holistic process, leaving no stone unturned....